mne_bids/dig.py

```diff
@@ -290,6 +291,23 @@ def _coordsystem_json(*, raw, unit, hpi_coord_system, sensor_coord_system,
             'iEEGCoordinateUnits': unit,  # m (MNE), mm, cm , or pixels
         }
 
+    # note that any coordsystem.json file shared within sessions
+    # will be the same across all runs (currently). So
+    # overwrite is set to True always
+    # XXX: improve later when BIDS is updated
+    # check that there already exists a coordsystem.json
+    if datatype == 'meg' and Path(fname).exists() and not overwrite:
+        with open(fname, 'r') as fin:
```

Comment:
We need to be explicit about the encoding, otherwise we'll run into issues on Windows
```suggestion
        with open(fname, 'r', encoding='utf-8-sig') as fin:
```